packages/perseus/src/components/sortable.tsx

```diff
@@ -397,12 +405,10 @@ class Draggable extends React.Component<DraggableProps, DraggableState> {
 export type SortableOption = string;
 
 type SortableProps = {
-    constraints:
-        | {
-              width: number;
-              height: number;
-          }
-        | Record<any, any>;
+    constraints: {
+        width?: number;
+        height?: number;
+    };
```

Comment:
This is a more accurate reflection of how the prop is used.  `Matcher` only sets the `height`.